content/en/docs/home/contribute/localization.md

```diff
@@ -75,9 +84,12 @@ Tutorials | [Kubernetes Basics](https://kubernetes.io/docs/tutorials/), [Hello M
 ## Next steps
```

Comment:
I'd lose this heading and put this all into a whatsnext capture.

Reply:
will do